scrapy/downloadermiddlewares/retry.py

```diff
@@ -70,8 +72,10 @@ def _retry(self, request, reason, spider):
             retryreq.meta['retry_times'] = retries
             retryreq.dont_filter = True
             retryreq.priority = request.priority + self.priority_adjust
+            self.stats.inc_value('retry/count')
```

Comment:
I don't mind having verbose stats if it's to debug problems. Having to look at the code for `EXCEPTIONS_TO_RETRY` and default `RETRY_HTTP_CODES`  (and settings to double check if it's not the default) feels not-straightforward.
I'm not sure I would spot `retry/count` more than high `downloader/exception_type_count` values.